devtools/maven/src/main/java/io/quarkus/maven/GenerateExtensionsJsonMojo.java

```diff
@@ -59,6 +64,9 @@
     @Parameter(property = "bomVersion", defaultValue = "${project.version}")
     private String bomVersion;
 
+    @Parameter(property = "overridesFile", defaultValue = "${project.basedir}/src/main/resources/extensions-overrides.json")
```

Comment:
this seems fishy  - users can ovveride the name of resources folder - isn't there a more maven agnostic way to look these up ?

Reply:
`${project.build.outputDirectory}` the resources will end up being there, also possibly filtered.

Reply:
Using `${project.build.outputDirectory}` doesn't seem to work. There's no output directory basically (there's just `target`, no `target/classes`)

Reply:
Before I was using `${project.resources[0].resource}` but that doesn't work. The expression is probably too complex.

Reply:
Ah, that's right.

Reply:
Also, why is this fishy? All paths are basically overridable, right? I mean is there some way this could be abused?

Reply:
ah this the defaultvalue...oh well then i don't care at this stage - maybe post 1.0 if ever ;)